<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pMainWindow</class>
 <widget class="QMainWindow" name="pMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="plabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_as_A"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_Q"/>
   </widget>
   <widget class="QMenu" name="menuCryption_C">
    <property name="title">
     <string>Cryption(&amp;C)</string>
    </property>
    <addaction name="actionEncryption_E"/>
    <addaction name="actionDecryption_D"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuCryption_C"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_O">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
  </action>
  <action name="actionSave_as_A">
   <property name="text">
    <string>Save as(&amp;A)</string>
   </property>
  </action>
  <action name="actionQuit_Q">
   <property name="text">
    <string>Quit(&amp;Q)</string>
   </property>
  </action>
  <action name="actionEncryption_E">
   <property name="text">
    <string>Encryption(&amp;E)</string>
   </property>
  </action>
  <action name="actionDecryption_D">
   <property name="text">
    <string>Decryption(&amp;D)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
